frontend/src/app/modules/auth/components/sign-in-form/sign-in-form.component.ts

```diff
@@ -46,9 +52,27 @@ export class SignInFormComponent extends BaseComponent {
     private handleAuthenticationResponse(resp: firebase.auth.UserCredential | void): void {
         if (resp) {
             this.userService
-                .getCurrentUser()
+                .createUser({
+                    uid: resp.user?.uid,
+                    userName: resp.user?.displayName ?? this.signInForm.get('name')?.value ?? '',
+                    email: resp.user?.email ?? '',
+                    image: resp.user?.photoURL ?? undefined,
+                    language: this.getLanguage(),
+                    timeFormat: this.getTimeFormat(),
+                    dateFormat: DateFormat.MonthDayYear,
+                    phone: resp.user?.phoneNumber ?? undefined,
+                    timeZone: new Date().getTimezoneOffset(),
+                })
```

Comment:
You should put repeating code to separate helpers. This code is same as code in sign up component